concepts/05 UI Components/DateBox/02 Accessibility/15 Keyboard Navigation.md

```diff
@@ -1,4 +1,4 @@
-An end user can use the following keys to interact with the UI component.
+A user can use the following keys to interact with the DateBox component: 
 
```

Comment:
Please update the following line below:

Line 22:

Moves focus to the day nearby.
Change to:
Moves focus to the nearest date.